apps/etterlatte-behandling/src/main/kotlin/inntektsjustering/AarligInntektsjusteringJobbService.kt

```diff
@@ -46,8 +55,19 @@ class AarligInntektsjusteringJobbService(
             if (!skalBehandlingOmregnes(sakId, loependeFom)) {
                 // TODO Legge til en begrunnelse
                 omregningService.oppdaterKjoering(KjoeringRequest(kjoering, KjoeringStatus.FERDIGSTILT, sakId))
-            } else if (kanIkkeKjoeresAutomatisk()) {
-                // TODO Finnes det tilfeller av dette?
+            } else if (!kanKjoeresAutomatisk(sakId)) {
+                // TODO bør opprette behandling for dette ikke bare oppgave..
```

Comment:
Er det nødvendig med en egen oppgavetype for de da? Vil de ikke naturlig ha en revurdering-oppgave knyttet til seg?

Reply:
Nei kanskje ikke. Holder kanskje at revurderingen er en egen årsak for dette.

Reply:
Fjernet det